### PR TITLE
Fix the missing parameter in the emitter interface.

### PR DESCRIPTION
**Pull request by stanleytakamatsu**

### CODE REVIEW

File: eventemitter2.d.ts [hunk 1]
```diff
@@ -11,12 +11,12 @@ declare module eventemitter2 {
          * @default '.'
          * @description the delimiter used to segment namespaces.
          */
-        delimiter?: string,
```
Comment (RangerMauve) [inline eventemitter2.d.ts]: Any insight as to what the changes in whitespace are?

Comment (stanleytakamatsu) [inline eventemitter2.d.ts]: The change is in the emitAsync method parameters.
Was missing the additional argument "values" in the method.

Comment (RangerMauve) [inline eventemitter2.d.ts]: Yeah, but it seems that there's more lines affected which have changes to the whitespace.

I guess it doesn't really matter.

**Comment by RangerMauve** [comment]
Out in 2.0.1. Thanks for the PR!

File: eventemitter2.d.ts (hunk 1 above)
Comment (stanleytakamatsu) [inline eventemitter2.d.ts]: Sorry, I use visual studio code, it should have automatically removed the whitespaces.

**Pull request merged**